lib/fireworq.go

```diff
@@ -6,6 +6,9 @@ import (
 	"fmt"
 	"net/http"
 	"strings"
+	"time"
+
```

Comment:
[nit] No need to insert a new line here.

Reply:
✅ https://github.com/fireworq/mackerel-plugin-fireworq/pull/1/commits/b579bd6d127c87dd18eb76cbc7adc16d453e00c3